RELEASE_NOTES.md

```diff
@@ -2,7 +2,7 @@
 
 - feat: `continue` statements. Like Dafny's `break` statements, these come in two forms: one that uses a label to name the continue target and one that specifies the continue target by nesting level. See section [19.2](https://dafny-lang.github.io/dafny/DafnyRef/DafnyRef#sec-break-continue) of the Reference Manual. (https://github.com/dafny-lang/dafny/pull/1839)
 - fix: No warning "File contains no code" if a file only contains a submodule (https://github.com/dafny-lang/dafny/pull/1840)
-
+- fix: export-reveals of function-by-method now allows the function body to be ghost
```

Comment:
```suggestion
- fix: export-reveals of function-by-method now allows the function body to be ghost (https://github.com/dafny-lang/dafny/pull/1855)

```
I believe that I links to the full PRs are useful, but we can also agree to skip them to avoid the noise.

Reply:
I like it to see links to PRs in the release notes.